README.md

```diff
@@ -114,7 +114,7 @@ The table below lists recommender models that are available in RecTools.
 - For getting recommendations `filter_viewed` and `items_to_recommend` options are available
 - For item-to-item recommendations use `recommend_to_items` method
 - For feeding user/item features to model just specify dataframes when constructing `Dataset`. [Check our tutorial](examples/4_dataset_with_features.ipynb)
-
+- Some models support recommendations for warm/cold users and items. Check [documentation](https://rectools.readthedocs.io/en/stable/features.html#models) for the details.
```

Comment:
Let's add a line to prove that it's also very easy. Something like:

For warm / cold inference just provide all required ids in `users` or `target_items` parameters of `recommend` or `recommend_to_items` methods and make sure you have features in the dataset for warm users/items. **Nothing else is needed, everything works out of the box**. Check [documentation](https://rectools.readthedocs.io/en/stable/features.html#models) to see which models support this scenarios.